src/include/distributed/version_compat.h

```diff
@@ -12,7 +12,7 @@
 #define VERSION_COMPAT_H
 
 #include "postgres.h"
-#include "commands/explain.h"
+#include <commands/explain.h>
```

Comment:
looks odd, can we go back to quotation notation ?

Reply:
TL;DR: Unfortunately we can't. There is a difference in the search path for quotes and the bracket signs.

We need to include the `commands/explain.h` from postgres explicitly here. By using brackets we force the pre-compiler to only look in the include directories passed to the compiler. With quotes the first directory it searches for the include is the directory relative to this file (`src/include/distributed/`). Since there is a `commands/explain.h` in that directory it will include that file instead of postgres' header file.

Reply:
can we rename the file so that there won't be a conflict ?